sqllexer_utils.go

```diff
@@ -240,3 +240,8 @@ func replaceDigits(input string, placeholder string) string {
 
 	return builder.String()
 }
+
+func trimQuotes(input string, delim string, closingDelim string) string {
+	replacer := strings.NewReplacer(delim, "", closingDelim, "")
```

Comment:
We could probably move the replacer to a global variable initialized via `func init()`? Not a big deal but it would save on the cost of building the replacer instance.

Reply:
I thought about that but the replace is a bit dynamic, which depends on the opening and closing delimiter, such as `"` or `[`|`]`. One way of doing that is create all possible replacers as global var and retrieve based on delimiter.

Reply:
Oh sorry, I missed that. Yeah, I guess you could keep a mapping of replacers by delimiters but it might be becoming more tedious than it's worth. I think it's fine as-is.